packages/main/src/plugin/configuration-registry.spec.ts

```diff
@@ -79,6 +89,24 @@ beforeEach(async () => {
   readFileMock.mockResolvedValue(JSON.stringify({}));
   cpSync.mockReturnValue(undefined);
 
+  // Setup DefaultConfiguration mock for the new functionality
+  const { DefaultConfiguration } = await import('./default-configuration.js');
+  vi.mocked(DefaultConfiguration).mockImplementation(
+    () =>
+      ({
+        getContent: vi.fn().mockResolvedValue({}),
+      }) as unknown as InstanceType<typeof DefaultConfiguration>,
+  );
+
+  // Setup LockedConfiguration mock for the new functionality
+  const { LockedConfiguration } = await import('./locked-configuration.js');
+  vi.mocked(LockedConfiguration).mockImplementation(
+    () =>
+      ({
+        getContent: vi.fn().mockResolvedValue({}),
+      }) as unknown as InstanceType<typeof LockedConfiguration>,
+  );
```

Comment:
Same remark as https://github.com/podman-desktop/podman-desktop/pull/14316/files#r2412842928, we don't do that.